pkg/plugins/golang/v4/scaffolds/api_test.go

```diff
@@ -0,0 +1,207 @@
+/*
```

Comment:
I don’t think we should scaffold those for the project.
Instead, we should think about proposing feature gates in a way that is easy to use.
We also need to be careful not to overload projects with too many helpers.
We really avoid to do that, since it is hard to keep maintained and support as for the end users understand 

See how I was looking for this work: 
 https://github.com/kubernetes-sigs/kubebuilder/issues/849#issuecomment-3084109006